src/language-js/parse/babel.js

```diff
@@ -39,8 +38,9 @@ const parseOptions = {
   ranges: true,
 };
 const recordAndTuplePlugin = ["recordAndTuple", { syntaxType: "hash" }];
+const v8intrinsicPlugin = "v8intrinsic";
 const pipelineOperatorPlugins = [
-  ["pipelineOperator", { proposal: "smart" }],
+  ["pipelineOperator", { proposal: "hack", topicToken: "%" }],
```

Comment:
`%` is used in official explainer ( https://github.com/js-choi/proposal-hack-pipes ). So we follow it.